### DISCUSSION TITLE
Welcome to Orchid Discussions!

### DISCUSSION THREAD

**Discussion by tabuna** [orchidsoftware/platform#1390] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by G3z** [comment]
Hi everyone, I'm coding from Italy.
I use Orchid for every Laravel project I can and sometimes fix bugs or propose new features for edge cases
My biggest Orchid  product ATM is a multi tenant e-commerce.
